src/spac/transformations.py

```diff
@@ -320,77 +325,47 @@ def rename_labels(adata, src_annotation, dest_annotation, mappings):
     mappings : dict
         A dictionary mapping the original annotation labels to
         the new labels.
-
-    Returns
-    -------
-    adata : anndata.AnnData
-        The updated Anndata object with the new column containing the
-        renamed labels.
-
-    Examples

```

Comment:
@fangliu117 , that was a good example, can you add it back?

Reply:
Yes, it was added to the latest commit [f1f8b37].